src/reference/01-General-Info/04-Bintray-For-Plugins.md

```diff
@@ -115,6 +111,28 @@ API key for future use.
 *NOTE: We have to do this before we can link our package to the sbt
 org.*
 
+### User Documentation
+
+You should add a few lines to your plugin's README file so users know how to modify their `project/plugins.sbt` in order to use your plugin.
+SBT plugins always use Ivy style patterns, so the resolver incantation is a bit verbose.
+As always, the first component in the resolver is just a human-friendly name for the resolver.
```

Comment:
It should be noted that if the plugin is synced to sbt plugin repo, additional resolver is not needed.